website/docs/guides/upgrade.html.markdown

```diff
@@ -192,6 +192,11 @@ When the ESXi software is not upgraded, the finalization stage will fail. There
 the NSX management plane, the Edge appliances and the NSX bits on the ESXi hosts, and postpone the ESXi OS upgrade to 
 a later time.
 
+### Upgrading Global Manager
```

Comment:
I would add a sample HCL for GM upgrade. I know it's very similar to the LM one, but users appreciate being able to do a cut and paste to get a template they can edit for their TF configuration